core/src/main/java/io/servicecomb/core/definition/classloader/MicroserviceClassLoader.java

```diff
@@ -24,27 +24,27 @@
 
   private final String appId;
 
-  private final String microserviceName;
```

Comment:
As the microserviceName is used for a while, we cannot change the name easily due to back compatible reason.

Reply:
From business perspective, I think it is clear that microserceName is an argument.
For the classloader, it is core part and be separating from the argument outside. 
Additionally, it already named MicroserviceClassloader.

If it is already a convention, I am fine.